lib/database_consistency/helper.rb

```diff
@@ -29,13 +29,18 @@ def connection_config(klass)
       end
     end
 
+    def project_models
+      ActiveRecord::Base.descendants.select do |klass|
+        project_klass?(klass)
+      end
+    end
```

Comment:
Hi!

There is a regression in this change if you defined a model like this:

```
(Class.new(ApplicationRecord) do
         self.table_name = 'temporary_table'
         self.abstract_class = true
       end) => %i[company_id],
```

I get:

```
An error occurred while loading ./spec/db/database_consistency_spec.rb.
Failure/Error: (@models ||= super).dup

NameError:
  wrong constant name #<Class:0x00007f1bc55a78b8>
  
# /usr/local/bundle/ruby/3.2.0/gems/database_consistency-1.7.17/lib/database_consistency/helper.rb:60:in `const_source_location'
# /usr/local/bundle/ruby/3.2.0/gems/database_consistency-1.7.17/lib/database_consistency/helper.rb:60:in `project_klass?'
# /usr/local/bundle/ruby/3.2.0/gems/database_consistency-1.7.17/lib/database_consistency/helper.rb:34:in `block in project_models'
# /usr/local/bundle/ruby/3.2.0/gems/database_consistency-1.7.17/lib/database_consistency/helper.rb:33:in `select'
# /usr/local/bundle/ruby/3.2.0/gems/database_consistency-1.7.17/lib/database_consistency/helper.rb:33:in `project_models'
```

Maybe also we can check that `klass` is not an abstract one?